src/weapons/driller/SludgePump.java

```diff
@@ -35,21 +35,21 @@ public class SludgePump extends Weapon {
 	* Class Variables
 	****************************************************************************************/
 	
-	private double regularShotDamage;
-	private double chargedShotDamage;
-	private double chargedShotFragmentDamage;
-	private int numFragmentsPerChargedShot;
-	private double chargedShotWindup;
-	private int ammoPerChargedShot;
-	private int carriedAmmo;
-	private int magazineSize;
+	private final double regularShotDamage;
+	private final double chargedShotDamage;
+	private final double chargedShotFragmentDamage;
+	private final int numFragmentsPerChargedShot;
+	private final double chargedShotWindup;
+	private final int ammoPerChargedShot;
+	private final int carriedAmmo;
+	private final int magazineSize;
 	protected double rateOfFire;
-	private double reloadTime;
-	private double projectileVelocity;
-	private double regularShotPuddleRadius;
-	private double chargedShotPuddleRadius;
-	private double corrosiveDoTDuration;
-	private double puddleLifetime;
+	private final double reloadTime;
+	private final double projectileVelocity;
+	private final double regularShotPuddleRadius;
+	private final double chargedShotPuddleRadius;
+	private final double corrosiveDoTDuration;
+	private final double puddleLifetime;
```

Comment:
I would prefer it if these variables were not changed to be `final`.

Reply:
I guess this was added when I was messing with IntelliJ. It was added by the IDE, I don't know exactly what "final" does.